### PR TITLE
Align description with the latest changes in the TG repo

### PR DESCRIPTION
**Pull request by heidivanparys**
Align the README, process description and issue template with the documentation in https://github.com/INSPIRE-MIF/technical-guidelines. See commit messages for more information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+The purpose of this repository is to manage the changes to the INSPIRE XML schemas.
 
-The official location of the INSPIRE schemas — to be used within the INSPIRE resources — is, and will remain, https://inspire.ec.europa.eu/schemas. This repository, which is setup within [Action 2.3](https://webgate.ec.europa.eu/fpfis/wikis/display/InspireMIG/Action+2.3+Simplification+of+INSPIRE+implementation) of the [INSPIRE MIWP 2021-2024](https://webgate.ec.europa.eu/fpfis/wikis/display/InspireMIG/INSPIRE+work+programme+2021-24), is only a means to facilitate the change management process of the INSPIRE schemas.
+The INSPIRE XML schemas can be categorized in two categories:
 
-Change proposals to the current version of the schemas can be proposed and discussed in the issue tracker of this repository. These change proposals are then subject to the specific governance and release process, whose workflow and actors involved are detailed in [this document](https://github.com/INSPIRE-MIF/application-schemas/blob/main/governance-release-process/process.md).
+* GML application schemas, derived from the UML appliation schemas
```
Comment (MarcoMinghini) [inline README.md:7]: there is a typo here in the word 'application'

File: README.md [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+* XML schemas used in the Extended Capabilities section of INSPIRE spatial data services based on Open Geospatial Consortium (OGC) WxS Services
+
+The official location of the INSPIRE XML schemas — to be used within the INSPIRE resources — is, and will remain, https://inspire.ec.europa.eu/schemas. This repository is only a means to facilitate the change management process of the INSPIRE schemas.
+
+## Governance and release process
+
+Change proposals to the current version of the schemas can be proposed and discussed in the issue tracker of this repository. These change proposals are then subject to the specific governance and release process, whose workflow and actors involved are detailed in [this document](/governance-release-process/process.md).
```
Comment (MarcoMinghini) [inline README.md:14]: maybe 'issue tracker' can be a link to the issue tracker.

**Review by MarcoMinghini — APPROVED** [review]
Thanks @heidivanparys for the changes. I spotted a couple of very minor things, the documents are totally ok. Well done!

File: README.md (hunk 1 above)
Comment (heidivanparys) [reply README.md]: Fixed.

**Comment by heidivanparys** [comment]
Thanks @MarcoMinghini, I fixed the things you mentioned.

**Pull request merged**